Fenced_Frames_Ads_Reporting.md

```diff
@@ -168,16 +167,15 @@ window.fence.setReportEventDataForAutomaticBeacons({
 });
 ```
 
+If `setReportEventDataForAutomaticBeacons` is invoked, the browser will send an automatic beacon to all registered URLs, but will only send an event data body (the information in `eventData`) with the HTTP request to destinations specified in the `destination` field.
```

Comment:
"... to all URLs registered via registerAdBeacon for the given event but will only send an event data body (the information in `eventData`) with the HTTP request to destinations specified in the `destination` field. This means that invoking `setReportEventDataForAutomaticBeacons` acts as an opt-in by the fenced frame document to allow sending the beacon to all registered URLs, aligning with cross-origin security principles."